schemas/context/media-timed-asset-reference.schema.json

```diff
@@ -21,11 +21,39 @@
           "format": "uri-reference",
           "description": "Identifier of the content, which can be used to tie back to other industry or CMS IDs."
         },
+        "dc:creator": {
+          "title": "Creator",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "An entity primarily responsible for making the resource."
+        },
+        "dc:publisher": {
+          "title": "Publisher",
+          "type": "string",
+          "description": "An entity responsible for making the resource available."
+        },
         "dc:title": {
           "title": "Media name",
           "type": "string",
           "description": "The friendly, human-readable name of the timed media asset."
         },
+        "xdm:recordLabel": {
```

Comment:
Isn't there a pre-existing media grammar that could be used?  Do we really need to invent a new one??

Reply:
Changed this with a reference to ID3.